### PR TITLE
README.md updated to explain SRV Record support.

### PR DESCRIPTION
**Pull request by ArtfulCoder**
#9073

### CODE REVIEW

File: cluster/addons/dns/README.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ Services, this resolves to the set of IPs of the pods selected by the Service.
 Clients are expected to consume the set or else use standard round-robin
 selection from the set.
 
+### SRV records
+SRV Records are created for Named ports that are part of "Normal" or Headless
```
Comment (thockin) [inline cluster/addons/dns/README.md]: no need to capitalize "Named"

no need to quote "Normal" here - it was explained above.

File: cluster/addons/dns/README.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ Services, this resolves to the set of IPs of the pods selected by the Service.
...
+Services.
+For each named port, the SRV record would have the form
+`_my-port-name._my-port-protocol.my-svc.my-namespace.svc.cluster.local`.
+For a regular service, this resolves to the port number and the CNAME:
+`my-svc.my-namespace.svc.cluster.local`.
+For a headless service, this resolves to the multiple answers, one for each pod
```
Comment (thockin) [inline cluster/addons/dns/README.md]: s/the multiple/multiple/

**Comment by ArtfulCoder** [comment]
incorporated feedback. PTAL

**Comment by thockin** [comment]
LGTM

**Pull request merged**